templates/frontend/pages/navigationMenuItemViewContent.tpl

```diff
@@ -0,0 +1,17 @@
+{**
+ * frontend/pages/navigationMenuItemViewContent.tpl
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2003-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * Display NavigationMenuItem content 
+ *}
+{include file="frontend/components/header.tpl" pageTitleTranslated=$title}
+
+<h2>{$title|escape}</h2>
```

Comment:
Use `<h1>` for the page title and nest it under the `<div class="page">` element.